common/util.c

```diff
@@ -139,6 +140,17 @@ bool parse_boolean(const char *boolean, bool current) {
 	return false;
 }
 
+float parse_float(const char *value) {
+	errno = 0;
+	char *end;
+	float flt = strtof(value, &end);
+	if (*end || errno) {
+		wlr_log(WLR_DEBUG, "Invalid float value %s, defaulting to FLT_MIN", value);
+		return FLT_MIN;
```

Comment:
NAN would be a better sentinel value than this.
Or change the signature to `bool parse_float(const char *value, float *ret)`.